teamA/lib/Views/login_page.dart

```diff
@@ -0,0 +1,189 @@
+import 'package:flutter/material.dart';
```

Comment:
Are these Anthony's changes that've not yet been merged?

Reply:
Yea I just reached into his branch and integrated it on mine.